scripts/ingests/2MASS/ingest-2mass.py

```diff
@@ -0,0 +1,102 @@
+from scripts.ingests.utils import *
+from astropy.table import Table, unique, setdiff
+from sqlalchemy import func
+
+
+SAVE_DB = False  # save the data files in addition to modifying the .db file
+RECREATE_DB = False  # recreates the .db file from the data files
+VERBOSE = True
+DATE_SUFFIX = 'Sep2021'
+
+db = load_simpledb(db_file='SIMPLE.db', RECREATE_DB=RECREATE_DB)
+
+## FUNCTIONS
+
+# QUERY SIMBAD TO GET 2MASS PHOTOMETRY
+def query_tmass(source_names):
+    Simbad.reset_votable_fields()
+    Simbad.add_votable_fields('typed_id')  # keep search term in result table
+    Simbad.add_votable_fields('flux(J)')
+    Simbad.add_votable_fields('flux_error(J)')
+    Simbad.add_votable_fields('flux_bibcode(J)')
+    Simbad.add_votable_fields('flux(H)')
+    Simbad.add_votable_fields('flux_error(H)')
+    Simbad.add_votable_fields('flux_bibcode(H)')
+    Simbad.add_votable_fields('flux(K)')
+    Simbad.add_votable_fields('flux_error(K)')
+    Simbad.add_votable_fields('flux_bibcode(K)')
+
+    print('2MASS query started')
+    result_table = Simbad.query_objects(source_names)
+    print('2MASS query complete')
+
+    # find indexes which contain 2MASS results
+    ind = result_table['FLUX_BIBCODE_J'] == '2003yCat.2246....0C'
+
+    tmass_phot = result_table['TYPED_ID', 'FLUX_J', 'FLUX_ERROR_J', 'FLUX_H','FLUX_ERROR_H', 'FLUX_K', 'FLUX_ERROR_K'][ind]
+    tmass_phot_unique = unique(tmass_phot, keys='TYPED_ID',keep='first')
+    print(len(tmass_phot), len(tmass_phot_unique))
+    return tmass_phot_unique
+
+#get list of all sources
+sources = db.query(db.Sources.c.source).astropy()
+
+# Use SIMBAD to 2MASS designations for all sources
+# tmass_designations = find_in_simbad(sources['source'], '2MASS J')
+tmass_desig_file_string = 'scripts/ingests/2MASS/2MASS_designations_'+DATE_SUFFIX+'.xml'
+# tmass_designations.write(tmass_desig_file_string, format='votable', overwrite=True)
+tmass_designations = Table.read(tmass_desig_file_string, format='votable')
```

Comment:
Like for the Gaia ingest script pull request: I don't think we need to include the XML file as part of the PR, it is sufficient that the script includes the instructions/calls on how to generate it.

Reply:
Agreed! It’s not part of the PR.